lib/src/usage.dart

```diff
@@ -18,109 +17,107 @@ import 'utils.dart';
 ///
 /// It builds the usage text up one column at a time and handles padding with
```

Comment:
Somewhere in this doc can you include a reference to [lineLength] with an explanation of its functionality, including what it means for it to be null? Now that we don't have the documentation from the Usage field it may not be completely clear.

Reply:
Done.